.travis.yml

```diff
@@ -0,0 +1,6 @@
+language: node_js
+node_js:
+  - "6"
+script: npm run postinstall
+script: node index.js
```

Comment:
How does this work in travis environment? As it just loads the server and waits for input, I'm not sure how travis knows when it's complete?